vector/src/main/java/im/vector/app/features/spaces/create/BetaWarningBottomSheet.kt

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (c) 2021 New Vector Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package im.vector.app.features.spaces.create
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import im.vector.app.core.platform.VectorBaseBottomSheetDialogFragment
+import im.vector.app.databinding.BottomSheetSpaceCreatePrivateWarningBinding
+
+class BetaWarningBottomSheet : VectorBaseBottomSheetDialogFragment<BottomSheetSpaceCreatePrivateWarningBinding>() {
+
+    interface InteractionListener {
+        fun betaWarningOnContinueAnyway()
+    }
+
+    var interactionListener: InteractionListener? = null
+
+    override fun getBinding(inflater: LayoutInflater, container: ViewGroup?) =
+            BottomSheetSpaceCreatePrivateWarningBinding.inflate(inflater, container, false)
+
+    override val showExpanded = true
+
+    override fun onDetach() {
+        interactionListener = null
```

Comment:
setting to null is done several times then?

Reply:
yes, it's defensive coding, but can remove

Reply:
see previous reply